modules/manage/partials/mountable-topics.adoc

```diff
@@ -1,19 +1,19 @@
 == Prerequisites
 
+
+ifndef::env-cloud[]
 [NOTE]
 ====
 include::shared:partial$enterprise-license.adoc[]
 ====
 
-ifndef::env-cloud[]
 . xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
 . Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
 endif::[]
 ifdef::env-cloud[]
 . xref:manage:rpk/rpk-install.adoc[Install `rpk`] to run cluster mount and unmount operations on the command line, or xref:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API and execute these operations programmatically.
 +
 If using the Cloud API, you'll make requests against Data Plane APIs. Make sure you xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[retrieve the Data Plane API URL] and use this URL to perform a mount or unmount operation.
```

Comment:
Ah, now that we're removing the bullet point regarding Tiered Storage for the cloud version we should change the formatting for the cloud prereqs. I think we could do either an unordered list, but only if "If using the cloud API" is a nested bullet after the main one, or we get rid of the listing altogether. I could go either way.

Reply:
What about this?
https://github.com/redpanda-data/docs/pull/894/commits/18ffdf964a904041244855f935e9d77bdc275432